clear/main.py

```diff
@@ -242,5 +242,14 @@ def clear_vlan_fdb(vlanid):
     command = 'fdbclear' + ' -v ' + vlanid
     run_command(command)
 '''
+
+#
+# 'line' command
+#
+@cli.command('line')
+@click.argument('linenum')
+def line(linenum):
+    return
```

Comment:
Please add a docstring briefly describing the command (which will also be used as the help message), and add a comment such as `TODO: Stub` in place of the body.